--- a/include/sp_func.h
+++ b/include/sp_func.h
@@ -63,34 +63,34 @@ int	SP_leave( mailbox mbox, const char *
 
 int	SP_multicast( mailbox mbox, service service_type, 
 		      const char *group,
-		      int16 mess_type, int mess_len, const char *mess );
+		      short mess_type, int mess_len, const char *mess );
 
 int	SP_scat_multicast( mailbox mbox, service service_type,
                            const char *group,
-			   int16 mess_type, const scatter *scat_mess );
+			   short mess_type, const scatter *scat_mess );
 
 int	SP_multigroup_multicast( mailbox mbox, service service_type,
 				 int num_groups,
 				 const char groups[][MAX_GROUP_NAME],
-				 int16 mess_type, int mess_len,
+				 short mess_type, int mess_len,
 				 const char *mess );
 
 int	SP_multigroup_scat_multicast( mailbox mbox, service service_type,
 				      int num_groups,
 				      const char groups[][MAX_GROUP_NAME],
-				      int16 mess_type,
+				      short mess_type,
 				      const scatter *scat_mess );
 
 int	SP_receive( mailbox mbox, service *service_type,
 		    char sender[MAX_GROUP_NAME], int max_groups,
 		    int *num_groups, char groups[][MAX_GROUP_NAME],
-		    int16 *mess_type, int *endian_mismatch,
+		    short *mess_type, int *endian_mismatch,
 		    int max_mess_len, char *mess );
 
 int	SP_scat_receive( mailbox mbox, service *service_type,
 			 char sender[MAX_GROUP_NAME], int max_groups,
 			 int *num_groups, char groups[][MAX_GROUP_NAME],
-			 int16 *mess_type, int *endian_mismatch,
+			 short *mess_type, int *endian_mismatch,
 			 scatter *scat_mess );
 
 /* get membership info from a message */
