Index: spread-src-4.1.0/examples/sp_time_memb.c
===================================================================
--- spread-src-4.1.0.orig/examples/sp_time_memb.c	2011-12-20 15:16:01.211148011 +0000
+++ spread-src-4.1.0/examples/sp_time_memb.c	2011-12-20 15:16:16.179147867 +0000
@@ -58,7 +58,7 @@
 char    sender[MAX_GROUP_NAME];
 int     num_groups;
 char    groups[MY_MAX_NUM_GROUPS][MAX_GROUP_NAME];
-int16   mess_type;
+short   mess_type;
 int     endian_mismatch;
 int     mess_len;
 char    mess[MY_MAX_MESS_SIZE];
