--- a/include/fl.h
+++ b/include/fl.h
@@ -59,7 +59,7 @@
 #define FL_MAX_SCATTER_ELEMENTS (MAX_CLIENT_SCATTER_ELEMENTS - 1)
 
 /* minimum message type a user is allowed to use - less than this is illegal */
-#define FL_MIN_LEGAL_MESS_TYPE ((int16) -32765)
+#define FL_MIN_LEGAL_MESS_TYPE ((short) -32765)
 
 #ifdef __cplusplus
 extern "C" {
@@ -81,33 +81,33 @@ int   FL_leave(mailbox mbox, const char 
 int   FL_flush(mailbox mbox, const char *group_name);
 
 int   FL_unicast(mailbox mbox, service serv_type, const char *group_name, 
-		 const char *recvr_name, int16 mess_type, int mess_len, const char *mess);
+		 const char *recvr_name, short mess_type, int mess_len, const char *mess);
 
 int   FL_scat_unicast(mailbox mbox, service serv_type, const char *group_name,
-		      const char *recvr_name, int16 mess_type, const scatter *scat);
+		      const char *recvr_name, short mess_type, const scatter *scat);
 
 int   FL_subgroupcast(mailbox mbox, service serv_type, const char *group_name,
 		      int num_recvrs, char recvr_names[][MAX_GROUP_NAME],
-		      int16 mess_type, int mess_len, const char *mess);
+		      short mess_type, int mess_len, const char *mess);
 
 int   FL_scat_subgroupcast(mailbox mbox, service serv_type, const char *group_name,
 			   int num_recvrs, char recvr_names[][MAX_GROUP_NAME],
-			   int16 mess_type, const scatter *scat);
+			   short mess_type, const scatter *scat);
 
 int   FL_multicast(mailbox mbox, service serv_type, const char *group_name,
-		   int16 mess_type, int mess_len, const char *mess);
+		   short mess_type, int mess_len, const char *mess);
 
 int   FL_scat_multicast(mailbox mbox, service serv_type, const char *group_name,
-			int16 mess_type, const scatter *scat_mess);
+			short mess_type, const scatter *scat_mess);
 
 int   FL_receive(mailbox mbox, service *serv_type, char *sender_name,
 		 int max_groups, int *num_groups, char group_names[][MAX_GROUP_NAME],
-		 int16 *mess_type, int *endian_mismatch, int max_mess_len,
+		 short *mess_type, int *endian_mismatch, int max_mess_len,
 		 char *mess, int *more_msgs);
   
 int   FL_scat_receive(mailbox mbox, service *serv_type, char *sender_name,
 		      int max_groups, int *num_groups, char group_names[][MAX_GROUP_NAME],
-		      int16 *mess_type, int *endian_mismatch, scatter *scat_mess, 
+		      short *mess_type, int *endian_mismatch, scatter *scat_mess, 
 		      int *more_msgs);
   
 int   FL_more_msgs(mailbox mbox);
