Index: spread-src-4.1.0/examples/fl_user.c
===================================================================
--- spread-src-4.1.0.orig/examples/fl_user.c	2011-12-20 15:13:17.723149569 +0000
+++ spread-src-4.1.0/examples/fl_user.c	2011-12-20 15:14:05.363149116 +0000
@@ -303,12 +303,12 @@
   char		sender[MAX_GROUP_NAME];
   char		target_groups[100][MAX_GROUP_NAME];
   group_id	*grp_id;
-  int32		*num_vs;
+  int		*num_vs;
   char		*vs_members;
   int		num_groups;
   int		num_bytes;
   int		service_type;
-  int16		mess_type;
+  short		mess_type;
   int		endian_mismatch;
   int		i;
   int		ret;
@@ -353,8 +353,8 @@
 	  num_bytes = 0;
 	  grp_id = (group_id *)&mess[num_bytes];
 	  num_bytes += sizeof( group_id );
-	  num_vs = (int32 *)&mess[num_bytes];
-	  num_bytes += sizeof( int32 );
+	  num_vs = (int *)&mess[num_bytes];
+	  num_bytes += sizeof( int );
 	  vs_members = &mess[num_bytes];
 
 	  printf("Received REGULAR membership for group %s with %d members, where I am member %d:\n",
