Index: guile-2.0.2/configure.ac
===================================================================
--- guile-2.0.2.orig/configure.ac	2011-08-14 16:46:28.247690765 +0100
+++ guile-2.0.2/configure.ac	2011-08-14 16:46:49.176690278 +0100
@@ -1512,7 +1512,7 @@
 
 if test "$cross_compiling" = "yes"; then
   AC_MSG_CHECKING(guile for build)
-  GUILE_FOR_BUILD="${GUILE_FOR_BUILD-guile}"
+  GUILE_FOR_BUILD="/usr/local/bin/guile"
 else
   GUILE_FOR_BUILD='this-value-will-never-be-used'
 fi   
